test/wdio/wdio-globals.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { browser } from '@wdio/globals';
+import AxeBuilder from '@axe-core/webdriverio';
+import { assert } from 'chai';
+import express from 'express';
+import { fixturesPath } from 'axe-test-fixtures';
+import { Server, createServer } from 'http';
+import testListen from 'test-listen';
+
+describe('@wdio/globals', () => {
+  let server: Server;
+  let addr: string;
+  let builder: AxeBuilder;
+  before(async () => {
+    const app = express();
+    app.use(express.static(fixturesPath));
+    server = createServer(app);
+    addr = await testListen(server);
+  });
+
+  beforeEach(() => {
+    // node v20 causes a typescript error due to our current type of WebdriverIO.Browser not being compatible with Browser from webriverio
+    builder = new AxeBuilder({ client: browser as any });
+  });
```

Comment:
NBD - reference https://github.com/dequelabs/axe-core-npm/issues/883 so we know to update it when this gets picked up